cmobase/background.adoc

```diff
@@ -7,17 +7,17 @@ This chapter provides information common to all CMO extensions.
 
 A _memory location_ is a physical resource in a system uniquely identified by a
 _physical address_. The _observers_ of a given memory location consist of the
-RISC-V harts or I/O devices that can access that memory location. A given
-observer may not be able to access all memory locations in a system, and two
-different harts or devices may or may not be able to observe the same set of
-memory locations.
+agents, such as RISC-V harts, accelerators, I/O devices, etc., that can access
+that memory location. A given observer may not be able to access all memory
+locations in a system, and two different agents may or may not be able to
+observe the same set of memory locations.
 
 In this specification, a _load operation_ (or _store operation_) is performed by
-an observer to consume (or modify) the data at a given memory location. For a
-RISC-V hart, a load or store operation may be performed as a result of an
-explicit or implicit memory access. Additionally, a _read operation_ (or _write
-operation_) is an operation performed on the memory location to fetch (or
-update) the data at the physical resource.
+an observer to consume (or modify) the data at a given memory location. Load or
+store operations may be performed as a result of explicit or implicit memory
```

Comment:
To reduce ambiguity, would recommend replacing "memory" in this sentence with "memory location", and restructuring the sentence slightly, with the following result:

Load or store operations may be performed as a result of explicit or implicit accesses to memory locations.

Reply:
I thought "explicit or implicit memory accesses" was a well understood term in the architecture, in particular with respect to executing load and store instructions. So I guess I'm not understanding the ambiguity...?

Reply:
The motivation behind the change is just to reduce the ambiguity of the term "memory."  Some computer architecture definitions for "memory" are more restrictive than the conveniently-broad definition of "memory location" that we create in this specification -- which, as I understand it, is intended explicitly to include physical resources that may not directly map to storage elements.

The intention of the original sentence is fairly clear - however,  since the preceding definition of "load operation" and "store operation" is defined with respect to "memory locations," it seemed clearest to continue that practice in this sentence.

Reply:
OK. I'll refer to the "memory location" in that sentence, and push things out.